crates/namada/src/ledger/protocol/mod.rs

```diff
@@ -363,22 +361,6 @@ where
     Ok(extended_tx_result)
 }
 
-/// Load the wasm hash for a transfer from storage.
-///
-/// # Panics
-/// If the transaction hash is not found in storage
-pub fn get_transfer_hash_from_storage<S>(storage: &S) -> Hash
```

Comment:
Ah this might have been my bad. I believe this was used to construct the tx for fee unshielding and I forgot to remove it, thanks!